dcos/marathon.py

```diff
@@ -290,7 +290,10 @@ def add_app(self, app_resource):
         else:
             app_json = app_resource
 
-        response = _http_req(http.post, url,
+        if not isinstance(app_json, list):
+            app_json = [app_json]
+
+        response = _http_req(http.put, url,
```

Comment:
Yes, though I see that as a separate PR. Atm nothing is done with the return value. It's just passed through.